misc/startup/keymaster-eventmond.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Keymaster event monitor server
+After=network.target
+
+[Service]
+ExecStart=/usr/local/sbin/keymaster-eventmond
+ExecReload=/bin/kill -HUP $MAINPID
+Restart=always
+RestartSec=1
+
```

Comment:
please make it run as a non-root user (user nobody is a good place to start)

Reply:
Done.